Program.cs

```diff
@@ -133,6 +135,117 @@ static async Task Main(string[] args)
             await skyManager.ApiClient.ExecuteRequestAsync(patchCard2Req); 
         }
 
+        private static async Task updateAndPatchCard(string command) {
+            string[] commandInfoList = command.Split("--"); // split based on the options   
+
+            string[] cardToUpdateData = commandInfoList[0].Split(" "); 
+            if(cardToUpdateData.Length < 2 || cardToUpdateData.Length > 3) {
+                Console.Error.WriteLine("Did not get correctly formatted required update information. Please verify your command and try again."); 
+                return; 
+            }
+            if(commandInfoList.Length < 2) {
+                Console.Error.WriteLine("No options specified. Please try again with at least one option"); 
+                return; 
+            }
+            Int32 cardPos = Int32.Parse(cardToUpdateData[1]); 
+
+            var getSeqReq = new Skylight.Api.Assignments.V1.SequenceRequests.GetSequenceRequest(assign.Id, assign.RootSequence); 
+            var seqResult = await skyManager.ApiClient.ExecuteRequestAsync(getSeqReq);
+            Sequence rootSeq = seqResult.Content; 
+            var cardsReq = new Skylight.Api.Assignments.V1.CardRequests.GetSequenceCardsRequest(assign.Id, assign.RootSequence); 
+            var cardsResult =  await skyManager.ApiClient.ExecuteRequestAsync(cardsReq);  
+            List<Card> cards = cardsResult.Content; 
+            Card card = cards[cardPos];
+
+            Dictionary<string, object> cardChanges = new Dictionary<string, object>(); 
+                    
+            // go through the provided options and make the appropriate changes 
+            for (int i= 1; i < commandInfoList.Length; i++) {
+                Console.WriteLine("Processing command: " + commandInfoList[i]); 
+                String[] optionDetails = commandInfoList[i].Split(" "); 
+
+                if(optionDetails.Length < 2) {
+                    Console.Error.WriteLine("Missing information in at least one option"); 
+                }
+
+                string option = optionDetails[0];
+                string newValue = optionDetails[1]; 
+            
+                if(option.Equals("selectable")) {
```

Comment:
This gets a bit spaghetti-esque here. I think it's fine for now; in the future it may be useful to have an extensible way of adding options with callbacks.

Reply:
You def right. Do you have any ideas for how to do that? I'd love to hear em since atm i can't think of anything better :(